app/src/main/res/layout/activity_weather_slider.xml

```diff
@@ -1,23 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
     <Button
         android:id="@+id/fetchApiDataButton"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="@string/fetch_api_data"
-        android:onClick="onFetchButtonTap"
         android:layout_alignParentBottom="true"
-        android:layout_centerHorizontal="true" />
+        android:layout_centerHorizontal="true"
+        android:onClick="onFetchButtonTap"
+        android:text="@string/fetch_api_data" />
 
     <TextView
-        android:id="@+id/summaryTextView"
+        android:id="@+id/temperatureTextView"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_alignParentTop="true"
         android:layout_centerHorizontal="true"
-        android:text="" />
+        android:text="temperatureTextView" />
```

Comment:
You don't want to have template text in your layout. Instead you should take advantage of the `tools` namespace

https://developer.android.com/studio/write/tool-attributes.html